src/main/java/wooteco/subway/auth/infrastructure/AuthorizationExtractor.java

```diff
@@ -1,12 +1,13 @@
 package wooteco.subway.auth.infrastructure;
 
-import javax.servlet.http.HttpServletRequest;
 import java.util.Enumeration;
 
+import javax.servlet.http.HttpServletRequest;
+
 public class AuthorizationExtractor {
     public static final String AUTHORIZATION = "Authorization";
-    public static String BEARER_TYPE = "Bearer";
     public static final String ACCESS_TOKEN_TYPE = AuthorizationExtractor.class.getSimpleName() + ".ACCESS_TOKEN_TYPE";
+    public static String BEARER_TYPE = "Bearer";
```

Comment:
```suggestion
    public static final String BEARER_TYPE = "Bearer";
```

제공된 코드지만 확인 부탁드려요~ 😉